### ISSUE TITLE
Questions from Code Coverage

### ISSUE THREAD

**Issue by Turtlelord26** [Zefalcon/ATSSG#49]
To follow below

**Comment by Turtlelord26** [comment]
What is UnitAction?

**Comment by Turtlelord26** [comment]
Are the various 'set' methods in the actions used in scripting, or are they just redundant with the constructors?

**Comment by Turtlelord26** [comment]
Confirm numTurns related methods in Action are dead code?

**Comment by Zefalcon** [comment]
UnitAction is an overarching superclass.  Not sure about the set methods at
the moment.  Probably used in giving units orders.

**Comment by Turtlelord26** [comment]
static int getNextId in Entity: used?

**Comment by jak266** [comment]
Please just leave the code alone. It's a seed for multiturn actions.

**Comment by Turtlelord26** [comment]
I'm not touching the code, that's the whole purpose of asking about things here.

**Comment by Turtlelord26** [comment]
Parameterless Constructor in AIConfig was never called.

**Comment by Turtlelord26** [comment]
TurnCounter seems like a dead class.

**Comment by Turtlelord26** [comment]
I did not manage to call lines 374 - 389 in GameMap's load method

**Comment by Turtlelord26** [comment]
Oh hey, ECLemma gives us package info:
Coverage by subpackage (recalling tests are counted against):
Main: 80.1%
Script: 42.5% (misleading)
Script.Framework: 4.9% (misleading)
Player: 43.8%
Player.AI: 51.8%
Enums: 74.5% (Buildings are to blame)
Actions: 43.1% (Buildings and resources)
GUI: 93.8%

**Comment by TomShkurti** [comment]
IIRC, there are some error statements in Load that should never be called.

**Comment by Turtlelord26** [comment]
Yes I didn't bother mentioning such code from any class. That bit doesn't look like an error condition.

**Comment by Turtlelord26** [comment]
For the record, GlobalMap and Overlay from GUI are dead classes.

**Comment by Turtlelord26** [comment]
Some misc get/set methods in entity and cell appear unused.
